tests/e2e/helpers/theme.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Helper method to check if user can click on theme button and toggle theme color
+ */
+export const canSwitchTheme = async () => {
+  const LIGHT_THEME_CLASS = 'bg-slate-50';
+  const DARK_THEME_CLASS = 'bg-gray-800';
+
+  const app = await $('.App').getElement();
+  const toggleThemeButton = await $('button=Toggle theme').getElement();
```

Comment:
I want to propose this:

```suggestion
  const TOGGLE_BUTTON_CONTENT = 'Toggle theme';

  const app = await $('.App').getElement();
  const toggleThemeButton = await $(`button=${TOGGLE_BUTTON_CONTENT}`).getElement();

```
because of the IDE error:
![image](https://github.com/user-attachments/assets/bcf14ad2-a5bc-4030-9c3f-b71fc92dfe6c)

It works on my machine

Refactor this statements in other tests.